fxa-client/ffi/src/lib.rs

```diff
@@ -349,3 +349,32 @@ define_box_destructor!(Config, fxa_config_free);
 define_box_destructor!(OAuthInfoC, fxa_oauth_info_free);
 define_box_destructor!(ProfileC, fxa_profile_free);
 define_box_destructor!(SyncKeysC, fxa_sync_keys_free);
+
+#[no_mangle]
+pub unsafe extern "C" fn _fns() -> Vec<*const u8> {
+    let mut fns = vec![
```

Comment:
FYI: This is missing off `fxa_str_free`, `fxa_free`, `fxa_config_free`, `fxa_oauth_info_free`, `fxa_profile_free`, and `fxa_sync_keys_free`, possibly more.

Gentle suggestion that we should either have some verification that this list is correct (like the megazord-symbol-checker in the other PR does, although it's certainly not the only way you could do it), or that it should be automatically generated.

Reply:
For the record: we're going to ensure that LTO is enabled, which exposes all of these `pub extern "C"` symbols, even on Android.